book/src/development/define_lints.md

```diff
@@ -0,0 +1,177 @@
+# Define New Lints
+
+The journey of any new Clippy lint begins with one step.
+And this first step is to define the lint and register it to the codebase.
+We can use the Clippy dev tools to handle this step since setting up the 
+lint involves some boilerplate code.
+
+## Name the Lint
+
+A wise software engineer once said:
+> There are only two hard things in Computer Science: cache invalidation and naming things.
+
+Naming a lint is no exception.
+Therefore, in case of uncertainty about if the name you chose fits the lint,
+please consult our [lint naming guidelines][lint_naming] or ask a maintainer
```

Comment:
> ask a maintainer

This is a good suggestion. In the general contribution docs, I would add a note that it's better to ping a [Clippy team member](https://www.rust-lang.org/governance/teams/dev-tools#Clippy%20team) directly. As questions in issues are sometimes overlooked. (This is not ideal, but sadly the truth, as reviewing every comment is very time consuming)